src/components/TaskRunMenu.vue

```diff
@@ -0,0 +1,106 @@
+<script>
```

Comment:
Not an issue just intrigued as I can't see it - where are we using this component?

Reply:
Ah yes - this component is related to your comment about a more robust popover; I haven't adapted it to the new data structure of the timeline but it's used in the gantt chart right now.